src/comm.cc

```diff
@@ -780,36 +781,41 @@ commCallCloseHandlers(int fd)
     }
 }
 
+/// sets SO_LINGER socket(7) option
+/// \param enabled -- whether linger will be active (sets linger::l_onoff)
+/// \param timeoutInSeconds -- how long to linger for (sets linger::l_linger)
+static void
+commConfigureLinger(const int fd, const OnOff enabled, const int timeoutInSeconds = 0)
+{
+    struct linger l;
+    l.l_onoff = (enabled == OnOff::on ? 1 : 0);
+    l.l_linger = timeoutInSeconds;
```

Comment:
All OnOff::on callers use zero timeoutInSeconds. It is difficult for me to justify keeping that parameter, especially when it has a problematic non-chrono type. If we do not plan to use it in upcoming PRs, let's drop timeoutInSeconds.

```suggestion
    l.l_linger = 0; // how long to linger for, in seconds
```

I realize that dropping third integer argument reduces the incentive to add OnOff, but I think we should still add it.

Reply:
Dropped the parameter.